tests/foreman/api/test_discoveredhost.py

```diff
@@ -381,53 +257,6 @@ def test_positive_auto_provision_pxe_host(
 
         :CaseImportance: Critical
```

Comment:
Is this importance appropriate? If it is, it shows that implementing it should be a high priority.

Reply:
I would say yes, it is critical to the component. I'm currently going through the tests I stubbed here in the API tests. This will be the next one after I get pxeless discovery working.